be/src/runtime/mem_tracker.h

```diff
@@ -267,7 +267,7 @@ class MemTracker {
     }
 
     void release_without_root(int64_t bytes) {
-        if (bytes == 0) {
+        if (bytes == 0 || _all_trackers.empty()) {
```

Comment:
under what condition, will this _all_trackers be empty? `MemTracker` init failed?